assets/elm/Decoder.elm

```diff
@@ -129,11 +128,13 @@ gameDecoder =
 type alias JoinTableState =
     { table : String
     , playerId : String
+    , topic : String
```

Comment:
table is now "what you tell your friends to enter in the 'join table' field", but it's no longer the actual topic string for Phoenix. That topic string is stored in `topic` now

A couple things this gets us:
1. We can prevent joining a game in progress because the hostess can "forget" about the names to games that have already started (while still maintaining unique topic strings for each table).
2. The hostess can re-use a table name as soon as that table's game has started.